README.md

```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 When you contribute, please take the following things into account:
 
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, run the following command:
```

Comment:
Could you please adapt the indentation to match the indentation of the previous lines?